server/migrations/20190406192726_companies_point_partners.ts

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+import * as Knex from 'knex';
+
+/* eslint-disable @typescript-eslint/camelcase */
+export async function up(knex: Knex) {
+  await knex.schema.createTable('company_point_partner', (table) => {
+    table.increments('id').primary();
+    table
+      .integer('user_id')
```

Comment:
Shouldn't this be camel case?

Reply:
I did this refactor first and then did the rename refactor.